src/colorizer/TimeControls.ts

```diff
@@ -81,22 +86,34 @@ export default class TimeControls {
     this.timerId = window.setTimeout(() => loadNextFrame(this.canvas.getCurrentFrame()), 0);
   }
 
-  public async handlePlayButtonClick(): Promise<void> {
+  /**
+   * Begins playback of the time series. If the time series is already playing, this function does
+   * nothing.
+   * @param onNewFrameCallback An optional callback that will be called whenever a new frame is loaded.
+   */
+  public async play(onNewFrameCallback: () => void = () => {}): Promise<void> {
```

Comment:
I don't see when this callback is passed in anywhere

Reply:
The callback isn't currently being used, yeah. I figured I should do the refactor to make it make it accessible in case that changes, but it's a maybe YAGNI situation :) Remove?